core/cmd/client.go

```diff
@@ -100,7 +102,7 @@ func (n ChainlinkAppFactory) NewApplication(cfg config.GeneralConfig) (chainlink
 	if err != nil {
 		return nil, err
 	}
-	keyStore := keystore.New(gormDB, utils.GetScryptParams(cfg))
+	keyStore := keystore.New(gormDB, utils.GetScryptParams(cfg), globalLogger.Named("KeyStore"))
```

Comment:
Would it make sense to push this `Named` call in to the constructor?

Reply:
Sure, if that's the pattern we are using.